articles/api-auth/tutorials/verify-access-token.md

```diff
@@ -2,14 +2,14 @@
 description: How an API can verify a bearer JWT access token
```

Comment:
Update the description metadata

Reply:
To what? That still seems correct for this document.